src/etc/lldb_lookup.py

```diff
@@ -115,4 +115,4 @@ def synthetic_lookup(valobj, dict):
     if rust_type == RustType.STD_REF_CELL:
         return StdRefSyntheticProvider(valobj, dict, is_cell=True)
 
-    return DefaultSynthteticProvider(valobj, dict)
+    return DefaultSyntheticProvider(valobj, dict)
```

Comment:
This is the only interesting part of this change. I'm assuming that uppercase doesn't imply a frozen/public symbol.